Contributing.md

```diff
@@ -45,7 +60,7 @@ Use Git to contribute substantive changes, such as:
 * Contributing large changes to text.
 * Adding new topics.
 
-### To Contribute using Git
+#### To Contribute using Git
```

Comment:
This also seems to be about editing articles when you need to submit an issue instead. maybe these two contribution sections should be removed?